native/cocos/renderer/pipeline/custom/NativeExecutor.cpp

```diff
@@ -259,10 +259,35 @@ PersistentRenderPassAndFramebuffer createPersistentRenderPassAndFramebuffer(
     data.renderPass = ctx.device->createRenderPass(rpInfo);
     fbInfo.renderPass = data.renderPass;
     data.framebuffer = ctx.device->createFramebuffer(fbInfo);
+    data.hash = gfx::Framebuffer::computeHash(fbInfo);
 
     return data;
 }
 
+PersistentRenderPassAndFramebuffer &fetchOrCreateFramebuffer(RenderGraphVisitorContext& ctx, const RasterPass& pass) {
+    auto iter = ctx.context.renderPasses.find(pass);
+    if (iter == ctx.context.renderPasses.end()) {
+        bool added = false;
+        std::tie(iter, added) = ctx.context.renderPasses.emplace(
+            pass, createPersistentRenderPassAndFramebuffer(ctx, pass));
+        CC_ENSURES(added);
+        return iter->second;
+    }
+
+    gfx::FramebufferInfo fbInfo{};
+    fbInfo.renderPass = iter->second.renderPass;
+    fbInfo.colorTextures = iter->second.framebuffer->getColorTextures();
+    fbInfo.depthStencilTexture = iter->second.framebuffer->getDepthStencilTexture();
+    auto hash = gfx::Framebuffer::computeHash(fbInfo);
+    if (iter->second.hash != hash) {
+        auto &res = ctx.context.renderPasses[pass];
+        res.framebuffer = ctx.device->createFramebuffer(fbInfo);
```

Comment:
`swapchain` updates backbuffer after surface re-created, but minimize and recover operation would not change width and height, so I recaculate hash here with `swapchain::generation`.